src/lang-go.ts

```diff
@@ -304,7 +306,7 @@ function mkSendRequest(address: string, token: string | undefined): Observable<S
         }
     }
 
-    const sendRequest: SendRequest = async ({ rootURI, requestType, request, useCache }) => {
+    const sendRequest: SendRequest<any> = async ({ rootURI, requestType, request, useCache }) => {
```

Comment:
The `any` here (and elsewhere in this file) doesn't make this less type safe than it was before.